index.js

```diff
@@ -225,6 +231,30 @@ function init (ssb, config) {
     },
 
     application: scuttle.application,
-    poBox: scuttle.poBox
+    poBox: scuttle.poBox,
+    subtribe: {
+      create (groupId, opts, cb) {
+        // create a new group
```

Comment:
```js
scuttle.poBox.create({}, (err, data) => {
  const { poBoxId } = data
})
```

Reply:
TODO 
- publish the poBox keys to the subgroup
  - `scuttle.group.create({ addPOBox: true }, cb)`